pkg/provider/models.go

```diff
@@ -52,6 +52,8 @@ type resolveRequest struct {
 	Apply             bool                   `json:"apply"`
 	EvaluationContext map[string]interface{} `json:"evaluation_context"`
 	Flags             []string               `json:"flags"`
+	SdkId             string                 `json:"sdk_id"`
```

Comment:
Think the [proto](https://ghe.spotify.net/konfidens/epx-flags-resolver/blob/master/epx-flags-resolver-lib/src/main/proto/confidence/flags/resolver/v1/api.proto#L90) for this looks different, or? 

so we would need to create a json payload that looks something like this: 
```
...other things
 "sdk": {
   "id": "<sdk_id>",
   "version": "<version>"
}
```

Reply:
You are right, I actually had this fix locally and verified it works end-to-end, but somehow forgot to force-push the change here. Good catch 👍